.taskcluster.yml

```diff
@@ -266,6 +266,14 @@ tasks:
                                       type: 'directory'
                                       path: '/builds/worker/artifacts'
                                       expires: {$fromNow: '1 year'}
+                                  'public/docker-contexts':
```

Comment:
This is needed since https://hg.mozilla.org/ci/taskgraph/rev/004bada4f170794a94ad43b84dd7562946305d22 was added, otherwise taskgraph breaks.